addons/dogtags/Dogtag.hpp

```diff
@@ -4,8 +4,8 @@ class RscStructuredText;
 class RscTitles {
     class GVAR(singleTag) {
         idd = -1;
-        onLoad = QUOTE(uiNamespace setVariable [ARR_2(QUOTE(QGVAR(tag)),_this select 0)]);
-        onUnload = QUOTE(uiNamespace setVariable [ARR_2(QUOTE(QGVAR(tag)),nil)]);
+        onLoad = QUOTE(uiNamespace setVariable [ARR_2(QQGVAR(tag),_this select 0)]);
```

Comment:
I didn't write the original code (_runs away_)! Although I dislike those set/getVariable macros.